docs/cmd/kn_service_list.md

```diff
@@ -31,6 +31,7 @@ kn service list [name] [flags]
       --allow-missing-template-keys   If true, ignore any errors in templates when a field or map key is missing in the template. Only applies to golang and jsonpath output formats. (default true)
   -h, --help                          help for list
   -n, --namespace string              List the requested object(s) in given namespace.
+      --no-headers                    Remove column headers in default table output.
```

Comment:
*When using the default output format, don't print headers (default print headers).